bin/pt-online-schema-change

```diff
@@ -11237,6 +11239,17 @@ sub find_renamed_cols {
       next if lc($orig_tbl) eq lc($new_tbl);
       $renames{lc($orig_tbl)} = $new_tbl;
    }
+
+   # Also handle RENAME COLUMN syntax (MySQL 8.0+)
+   while ( $alter =~ /$alter_rename_col_re/g ) {
+      my ($orig, $new) = map { $tp->ansi_to_legacy($_) } $1, $2;
+      next unless $orig && $new;
+      my (undef, $orig_tbl) = Quoter->split_unquote($orig);
+      my (undef, $new_tbl)  = Quoter->split_unquote($new);
+      # Silly but plausible: RENAME COLUMN same_name TO same_name ...
+      next if lc($orig_tbl) eq lc($new_tbl);
+      $renames{lc($orig_tbl)} = $new_tbl;
```

Comment:
I can only suggest an improvement here. We are doing lower case several times. 
It might be possible to introduce new variables and work with them. Here is the example of the change:

```
my $orig_tbl_lc = lc($orig_tbl);
my $new_tbl_lc  = lc($new_tbl);
next if $orig_tbl_lc eq $new_tbl_lc;
$renames{$orig_tbl_lc} = $new_tbl;
```